docs/tutorials/erc20-transfer-notifications/receive-email-when-there-is-an-erc20-transfer.md

```diff
@@ -124,7 +123,14 @@ const MESG = require('mesg-js').application()
 Then simply add the `whenEvent` function at the end of your code to link the event to the task:
 
 ```javascript
-MESG.whenEvent(erc20Transfer, sendEmail)
+MESG.listenEvent(erc20Transfer)
+  .on('data', (event) => {
+    const transfer = JSON.parse(event.eventData)
+    console.log('New ERC20 transfer received. will send an email. Transaction hash:', transfer.transactionHash)
+    MESG.executeTask(emailTask(transfer)).catch((err) => console.log(err.message))
```

Comment:
Let's put the code of `emailTask(transfer)` in.
It's was logic with the previous lib to extract event and task definition, but it's not anymore

Reply:
done